src/assets/signRequest.svg

```diff
@@ -1 +1 @@
-<svg width="141" height="141" fill="none" xmlns="http://www.w3.org/2000/svg"><circle cx="70.684" cy="70.969" r="69" stroke="url(#paint0_linear)" stroke-width="2"/><path d="M95.94 55.928v11.89L71.357 54.05 44.452 69.14v15.576l26.904 15.02 37.44-21v11.96l-37.44 21-37.581-21V63.16l37.51-21.07 24.656 13.838z" fill="url(#paint1_linear)"/><path d="M46.7 71.573v11.96l24.656 13.7 37.51-20.931V48.766l-37.51-21.07-37.581 21.07v11.89l37.51-21.07 26.834 15.09v15.646l-26.763 15.02L46.7 71.573z" fill="url(#paint2_linear)"/><defs><linearGradient id="paint0_linear" x1="161.101" y1="25.274" x2="70.684" y2="140.969" gradientUnits="userSpaceOnUse"><stop stop-color="#FFAA75"/><stop offset="1" stop-color="#FF9690" stop-opacity="0"/></linearGradient><linearGradient id="paint1_linear" x1="33.775" y1="99.597" x2="117.631" y2="48.561" gradientUnits="userSpaceOnUse"><stop stop-color="#FFC58F"/><stop offset="1" stop-color="#FF7C7C"/></linearGradient><linearGradient id="paint2_linear" x1="33.775" y1="99.597" x2="117.631" y2="48.561" gradientUnits="userSpaceOnUse"><stop stop-color="#FFC58F"/><stop offset="1" stop-color="#FF7C7C"/></linearGradient></defs></svg>
```

Comment:
I'm not sure, but we may want to change this icon. I think it's still using the old ANJ icon?

Reply:
OH yes good catch! Already asked @dizzypaty if we have replacements for these.